### PR TITLE
added functionality for additional instructions before adding courses…

### PR DESCRIPTION
**Pull request by lauradobson**
… and submitting ta application

### CODE REVIEW

**Review by amyleblond — DISMISSED** [review]
"final confirmation" - thumbs up :)

File: taworks/taform/templates/taform/application.html [hunk 1]
```diff
@@ -211,7 +211,13 @@ <h3>Confirmation</h3>
 <p><b>To submit your application you must agree to the above terms.</b></p>
 
 <p id="p01">Please note you will not be able to make changes to your application once you submit.<p>
-<input type="submit" id ="btn" value="Submit" disabled>
+<input type="hidden" name="btn" value="Submit" />
+<input type="button" id ="btn" value="Submit" disabled>
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: remove this if we aren't using it

I think we should also rename it from btn -> application-btn

**Review by j2kan — COMMENTED** [review]
@lauradobson 
The following task was also on the ticket:
 - Application Page - Add what the reason for applying to each course should contain

**Comment by j2kan** [comment]
@lauradobson @amyleblond 
if i press submit, then cancel then submit then submit the page just refreshes.

**Comment by lauradobson** [comment]
I wasn't able to replicate your submit, then cancel then submit submit error. @amyleblond  can you give it a try when you have a chance?

**Comment by lauradobson** [comment]
@j2kan if this happend when you had empty fields or error checks failed, the page will go back to the TA form and you will see error messages such as (which to my understanding is how the system should act):

image


Can you try this again with all the fields properly filled out and let me know if you get the same error? :)

**Comment by j2kan** [comment]
@lauradobson fyi: i was using chrome.

**Comment by lauradobson** [comment]
@j2kan I'm using chrome too

**Comment by j2kan** [comment]
@lauradobson @amyleblond  
maybe it's just bugging on my end =x

**Review by j2kan — CHANGES_REQUESTED** [review]
1. remove extra button line
2. add reason instructions

**Comment by amyleblond** [comment]
I had to run on Safari because my chrome isn't updating -- but I got the same issue as @j2kan 

1. fill out application form
2. press submit -> on pop up press cancel
3. don't make any changes to the form
4. press submit -> on pop up press submit

outcome: page is refreshed, all fields are clear and it hasn't been added to db

**Pull request closed without merging**